### PR TITLE
Update newsletter/firefox page (issue #15075)

### PR DESCRIPTION
**Pull request by alexgibson**
## One-line summary

> [!IMPORTANT]
> This needs to be live before Monday 16th Sept

- Updates the `/newsletter/firefox/` page with simplified layout and new copy.
- Adds A/B header image test for en-US, de, and fr languages.

## Issue / Bugzilla link

#15075

## Testing

Version A (fox tail image):

http://localhost:8000/en-US/newsletter/firefox/?v=1 (en-US)
http://localhost:8000/en-GB/newsletter/firefox/ (en-GB)
http://localhost:8000/de/newsletter/firefox/?v=1 (de)
http://localhost:8000/fr/newsletter/firefox/?v=1 (fr)
http://localhost:8000/es-ES/newsletter/firefox/ (es-ES)
http://localhost:8000/it/newsletter/firefox/ (it)
http://localhost:8000/nl/newsletter/firefox/ (nl)
http://localhost:8000/pl/newsletter/firefox/ (pl)
http://localhost:8000/el/newsletter/firefox/ (other locales with fallback copy)

Version B (alternate images):

http://localhost:8000/en-US/newsletter/firefox/?v=2 (en-US)
http://localhost:8000/de/newsletter/firefox/?v=2 (de)
http://localhost:8000/fr/newsletter/firefox/?v=2 (fr)

Experiment URL:

http://localhost:8000/en-US/newsletter/firefox/

- [x] Redirects to experiment variation for en-US, de, and fr (test with DNT/GPC disabled)

### CODE REVIEW

**Review by stephaniehobson — CHANGES_REQUESTED** [review]
We've changed how we record experiment data for GA4, please update to use the new syntax. (also, if you copy and pasted this code from somewhere, we should file an issue to update that too).

Also, there was a problem getting it running locally for me. Not sure if it was me or if you'll see it to if you stop and then re-start your dev server.

Oh... and it needs a rebase ;)

File: bedrock/newsletter/urls.py [hunk 1]
```diff
@@ -43,7 +43,14 @@
     path("newsletter/opt-out-confirmation/", views.recovery, name="newsletter.opt-out-confirmation"),
     # Branded signup pages for individual newsletters
     page("newsletter/mozilla/", "newsletter/mozilla.html", ftl_files=["mozorg/newsletters"]),
-    page("newsletter/firefox/", "newsletter/firefox.html", ftl_files=["mozorg/newsletters"]),
+    # Firefox newsletter A/B test. See issue 15075
+    path(
+        "newsletter/firefox/",
+        VariationTemplateView.as_view(
```
[the remark this answers is not in the document]
Comment (alexgibson) [reply bedrock/newsletter/urls.py:49]: Ah, looks like the line was removed in https://github.com/mozilla/bedrock/pull/15119/files#diff-05181ef311764e79e4e0f540afdb3d341a0294fed3347b5c4386a810b73c0783

I'll rebase and add it back here, thanks!

**Comment by alexgibson** [comment]
@stephaniehobson updated thanks

**Review by stephaniehobson — APPROVED** [review]
R+ 🦊

**Pull request merged**